src/customer/report.mustache

```diff
@@ -1624,14 +1624,17 @@ Below are the details about the templates used during PCA with <<Acronym>>. The
 \end{center}
 \hangindent1em \textbf{Key characteristics:} <<Level-1-Key_Characteristics>>
 
+%CISA averages rates should be updated yearly with previous year's average rates. Current rates are for 2020.
```

Comment:
For readability, we prefer comments with a space after the comment character and keeping lines shorter than 80 characters whenever possible:
```suggestion
% CISA averages rates should be updated yearly with previous year's average rates.
% Current rates are for 2020.
```